hil/cli/node.py

```diff
@@ -44,16 +44,31 @@ def node_bootdev(node, bootdev):
 
 @node.command(name='register', short_help='Register a new node')
 @click.argument('node')
+@click.argument('obmd-uri')
+@click.argument('obmd-admin-token')
 @click.argument('obmtype')
 @click.argument('hostname')
 @click.argument('username')
 @click.argument('password')
-def node_register(node, obmtype, hostname, username, password):
+def node_register(node,
+                  obmd_uri,
+                  obmd_admin_token,
```

Comment:
The combination is probably comparable to the current one, no? between obmtype, hostname, username, and password, node_register already requires a fair amount of typing. Remember obmd_uri & admin_token are eventually going to *replace* those, so I think it will end up being about the same.

Reply:
Ah right, I forgot that the other options are going away.